Rakefile

```diff
@@ -1,37 +1,39 @@
 # frozen_string_literal: true
 
-# Migrate
+require "sequel"
 
-migrate = lambda do |env, version|
+# Migrate
+migrate = lambda do |env, version, db_override: nil|
```

Comment:
just the `db_override` feels weird in the rakefile. Is it possible to maybe replace it with `db_override_for_tests` and name the variable inside the block to simply `db`